client.go

```diff
@@ -188,7 +188,7 @@ var previewnetNodes = map[string]AccountID{
 	"20.150.136.89:50211":           {Account: 9},
 }
 
-var mainnetMirror = []string{"hcs.mainnet.mirrornode.hedera.com:5600"}
+var mainnetMirror = []string{"mainnet-public.mirrornode.hedera.com:443"}
```

Comment:
This is a breaking change to customers using the old configuration. Mainnet public is still rate limited.  We shouldn't do this change without discussing it with leadership and rolling it out for all. Let's undo for now.